### ISSUE TITLE
Excessive dependency

### ISSUE THREAD

**Issue by akzhan** [adamwiggins/rush#1]
We use Ruby and Passenger for nginx in our projects.
No Mongrel at all.

So we need Rush gem without mongrel dependency.

Is it possible?

**Comment by trans** [comment]
Yes, this would be good for me too. How about making it an optional dependency? I.e. don't add it to gem dependencies and just raise an error if it is not available when attempted to use.

**Comment by AlekSi** [comment]
Isn't it fixed in 5b74839911600d6d7d5f5505bcfe6365d50c2970 ?

**Comment by al6x** [comment]
+1, 
ideally it shouldn't launch ruby process on remote machine at all and using only ssh
